added refinement for some unclear steps in the guide (#902)
port-labs/port-docs: docs/guides-and-tutorials/scaffold-a-new-service.md

1. First, let's create the necessary token and secrets:

> - Go to your [Github tokens page](https://github.com/settings/tokens), create a personal access token (classic) with `repo`, `admin:repo_hook` and `admin:org` scope, and copy it for the next step (this token is needed to create a repo from our workflow).

Reviewer: ```suggestion
- Go to your [Github tokens page](https://github.com/settings/tokens), create a personal access token (classic) with `repo`, `admin:repo_hook` and `admin:org` scope, and copy it (this token is needed to create a repo from our workflow).
```

  - To find the Bitbucket project key, go to `https://bitbucket.org/YOUR_BITBUCKET_WORKSPACE/workspace/projects/`, find the desired project in the list, and copy the value seen in the `Key` column in the table
:::

> 1. This page provides details about the action run. As you can see, the backend returned `Success` and the repo was successfully created (it can take a moment to reach `Success`):

Reviewer: ```suggestion
1. This page provides details about the action run. As you can see, the backend returned `Success` and the repo was successfully created (this can take a few moments):
```
